orion/benchmark.py

```diff
@@ -105,19 +105,25 @@ def _evaluate_signal(pipeline, name, dataset, signal, hyperparameter, metrics,
             name: scorer(truth, anomalies, test)
             for name, scorer in metrics.items()
         }
-        scores['elapsed'] = elapsed.total_seconds()
         scores['status'] = 'OK'
 
     except Exception as ex:
         LOGGER.exception("Exception scoring pipeline %s on signal %s (Holdout: %s), error %s.",
                          name, signal, holdout, ex)
 
+        elapsed = datetime.utcnow() - start
```

Comment:
In some cases, `start` may be undefined. For example, the exception came from load_pipeline.